src/layouts/Base.astro

```diff
@@ -18,6 +17,10 @@ const canonicalURL = new URL(Astro.url.pathname, Astro.site);
 const { title = t("site.title"), description = t("site.description"), image = '/img/wm-social.png', hasFooter = true } = Astro.props;
 ---
 <html lang={lang} data-theme="light">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap" rel="stylesheet">
+
```

Comment:
We can replace this with an inline style with the content of `https://fonts.googleapis.com/css2?` URL. Avoids one request to `fonts.googleapis.com`, and only needs requests to `fonts.gstatic.com`.

Although slightly more verbose (can extract it to a component), and harder to get automatic updates.

Ideally, we'd use something like [fontsource](https://github.com/fontsource/fontsource) to self-host these fonts. Avoids requests to Google trackers as well (GDPR). Can do as a follow up.